draft-ietf-add-ddr.md

```diff
@@ -411,12 +411,13 @@ resolvers with a large number of referring IP addresses.
 
 Clients MUST NOT use "resolver.arpa" as the server name either in the TLS
 Server Name Indication (SNI) ({{?RFC8446}}) for DoT, DoQ, or DoH connections,
-or in the URI host for DoH requests.
+or in the URI host for DoH requests.  The SNI extension is omitted when
+connecting to a server that is identified by an IP address.
 
 When performing discovery using resolver IP addresses, clients MUST
 use the IP address as the URI host for DoH requests.
 
-Note that since IP addresses are not supported by default in the TLS SNI,
```

Comment:
I think this was written this way because someone had defined some SNI name_type value for an IP address — I need to find the doc that defined that...